woocommerce/wc-admin
# Remove facebook extension from onboarding extensions fallback list

Fixes #7282

This PR removes Facebook marketing extension from the fallback free extensions list. Context: p6q8Tx-25X-p2#comment-5502

### Detailed test instructions:

- Set `remote-extensions-list` to `false` in `config/development.json` and rebuild app.
- Go to the OBW and look at the Free features tab
- Observe no Facebook extension in the list

### Screenshots:
![image](https://user-images.githubusercontent.com/3747241/124216258-67b4a980-db28-11eb-80e5-d309aaca747b.png)

Edit: No changelog after Jetpack changelogger is added
